.eslintrc.json

```diff
@@ -73,7 +73,8 @@
     "unicorn/no-null": "off",
     "prefer-regex-literals": "off",
     "class-methods-use-this": "off",
-    "no-underscore-dangle": "off"
+    "no-underscore-dangle": "off",
+    "import/no-extraneous-dependencies": ["error", { "devDependencies": true }]
```

Comment:
What is this changed for?

Reply:
It was due to the @types/uuid stating that it _shouldnt_ be a devDepency. This line and uuid have both been removed.